addons/uh60_engine/functions/fnc_perFrame.sqf

```diff
@@ -9,15 +9,90 @@
 #include "defines.hpp"
 params ["_vehicle", "_frameTime"];
 
```

Comment:
Do we need a locality check? Can this run on both pilot and copilot or should it just run on one at a time